fiftyone/utils/sam3.py

```
@@ -0,0 +1,537 @@
+            rm_op_mode = True
+        self.config.get_item_args["use_numpy"] = (
+            self.config.operation_mode == "visual"
+        )
+        get_item = super().build_get_item(field_mapping=field_mapping)
+        if rm_text_prompts:
+            _ = self.config.get_item_args.pop("text_prompts")
+        if rm_op_mode:
+            _ = self.config.get_item_args.pop("operation_mode")
+        return get_item
+
+    def _predict_all(self, args):
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `_predict_all`

(ANN202)